DemoApp/Shared/Token+Development.swift

```diff
@@ -26,7 +26,7 @@ extension StreamChatWrapper {
 
                 let numberOfSuccessfulRefreshes = refreshDetails.numberOfSuccessfulRefreshesBeforeFailing
                 let shouldNotFail = numberOfSuccessfulRefreshes == 0
-                if shouldNotFail || self.numberOfRefreshTokens <= numberOfSuccessfulRefreshes {
+                if shouldNotFail || self.numberOfRefreshTokens >= numberOfSuccessfulRefreshes {
```

Comment:
why has this changed?

Reply:
This was not working at all, I don't remember when I changed this, but I broke it. 

Glad you made this comment, because I forgot about adding this scenario to the QA Scenarios.

Reply:
Done